src/API/Helix/User/HelixFollow.ts

```diff
@@ -0,0 +1,35 @@
+import Twitch from '../../../';
+import { NonEnumerable } from '../../../Toolkit/Decorators';
+import HelixPagination from '../HelixPagination';
+import { UserIdResolvable } from '../../../Toolkit/UserTools';
+
+export interface HelixFollowFilter extends HelixPagination {
+	from?: UserIdResolvable;
+	to?: UserIdResolvable;
+}
+
+export interface HelixFollowData {
+    from_id: string;
+    to_id: string;
+    followed_at: string;
+}
+
+export default class HelixFollow {
+	@NonEnumerable protected _client: Twitch;
+
+    constructor(protected _data: HelixFollowData, client: Twitch) {
+        this._client = client;
+    }
+
+    get followedAt() {
+        return Date.parse(this._data.followed_at);
+    }
+
+    getTo() {
```

Comment:
Please use getters and name this method and the one after it in a more understandable way. Also, while you're at it, I'd swap the order of these two.
The indentation of this file also seems off. Please use tabs only.

Reply:
Changing the vocabulary would also mean changing it in the request params?

Reply:
Yup.